src/Symfony/Component/Mailer/Transport/AbstractHttpTransport.php

```diff
@@ -27,6 +27,7 @@ abstract class AbstractHttpTransport extends AbstractTransport
     protected $host;
     protected $port;
     protected $client;
+    protected $timeout = 5;
```

Comment:
this should default to PHP's `default_socket_timeout` ini setting

Reply:
`5` is also the default for SocketStream currently ... should we change it there too ?

Reply:
would make sense to me